console/src/pages/organizations/ListOrganizationsPage.tsx

```diff
@@ -108,10 +157,90 @@ export const ListOrganizationsPage = () => {
           variant="outline"
           onClick={() => fetchNextPage()}
         >
-          {isFetchingNextPage && <LoaderCircleIcon className="h-4 w-4 animate-spin" />}
+          {isFetchingNextPage && (
+            <LoaderCircleIcon className="h-4 w-4 animate-spin" />
+          )}
           Load more
         </Button>
       )}
     </div>
   );
 };
+
+const CreateOrganizationButton: FC = () => {
+  const navigate = useNavigate();
+
+  const form = useForm<z.infer<typeof organizationSchema>>({
+    resolver: zodResolver(organizationSchema),
+    defaultValues: {
+      displayName: '',
+    },
+  });
+
+  const createOrganizationMutation = useMutation(createOrganization);
+
+  const [open, setOpen] = useState(false);
+
+  const handleSubmit = async (values: z.infer<typeof organizationSchema>) => {
+    const createOrganizationResponse =
+      await createOrganizationMutation.mutateAsync({
+        organization: {
+          ...values,
+        },
+      });
+    toast.success('Organization created successfully');
+
+    setOpen(false);
+
+    navigate(`/organizations/${createOrganizationResponse.organization?.id}`);
+  };
+
+  return (
+    <AlertDialog open={open} onOpenChange={setOpen}>
+      <AlertDialogTrigger asChild>
+        <Button variant="outline">
+          <CirclePlus />
+          Create Organization
+        </Button>
+      </AlertDialogTrigger>
+      <AlertDialogContent>
+        <AlertDialogHeader>
+          <AlertDialogTitle>Create Organization</AlertDialogTitle>
+          <AlertDialogDescription>
+            Create a new organization.
+          </AlertDialogDescription>
+        </AlertDialogHeader>
+
+        <Form {...form}>
+          <form
+            className="space-y-4"
+            onSubmit={form.handleSubmit(handleSubmit)}
+          >
+            <FormField
+              control={form.control}
+              name="displayName"
+              render={({ field }) => (
+                <FormItem>
+                  <FormLabel>Display Name</FormLabel>
+                  <Input placeholder="ACME Corp" {...field} />
+                  <FormDescription>
+                    The display name of the organization. This will be displayed
+                    to users during the login process.
+                  </FormDescription>
+                </FormItem>
+              )}
+            />
+            <AlertDialogFooter className="mt-8">
+              <AlertDialogCancel>Cancel</AlertDialogCancel>
+              <Button type="submit">Save</Button>
+            </AlertDialogFooter>
+          </form>
+        </Form>
+      </AlertDialogContent>
+    </AlertDialog>
+  );
+};
+
+const organizationSchema = z.object({
+  displayName: z.string(),
+});
```

Comment:
can this be called `schema`? And either way, go before the CreateOrganizationButton?